multimodal-search-tirg/TirgImageEncoder/__init__.py

```diff
@@ -37,7 +37,7 @@ def post_init(self):
                 texts = pickle.load(fp)
             self.model = TIRG(texts, 512)
             model_sd = torch.load(self.model_path, map_location=torch.device('cpu'))
-            self.model.load_state_dict(model_sd['model_state_dict'])
+            self.model.load_state_dict(model_sd['model_state_dict'], strict=False)
```

Comment:
what does it do? just curious?

Reply:
@JoanFM I uncounted the same error as described [here](https://discuss.pytorch.org/t/missing-keys-unexpected-keys-in-state-dict-when-loading-self-trained-model/22379) while initializing pretrained tirg model.

Reply:
We were not facing this problem in the fashion model. Not sure why is it coming in CSS model. What happens when it tried to load weight from the state_dict but doesn't find the key. Does it initialise randomly? The documentation says 
"Whether you are loading from a partial state_dict, which is missing some keys, or loading a state_dict with more keys than the model that you are loading into, you can set the strict argument to False in the load_state_dict() function to ignore non-matching keys."